lib-object-oriented/line.js

```diff
@@ -1,34 +1,77 @@
 import Tag from './tag'
 
 export default class Line {
-  constructor( content ) {
+  constructor({ content, type }) {
     this.content = content
+    this.tagType = type
+    this.symbolCount = 0
+    this.getSymbolCount()
+    this.newLinesToSpaces()
+  }
+
+  newLinesToSpaces() {
+    const tagType = this.tagType,
+          content = this.content,
+          symbolCount = this.symbolCount
+
+    const findNewLine = element => {
+      return element.type === 'new_line'
+    }
+
+    if ( tagType !== '`' || ( tagType === '`' && symbolCount === 1 ) ) {
+      while ( content.findIndex( findNewLine ) > -1 ) {
+        const newLine = content.find( findNewLine )
+        if ( newLine ) {
+          newLine.type = ' '
+          newLine.value = ' '
+        }
+      }
+    }
+  }
+
+  getSymbolCount() {
```

Comment:
Every time `getSymbolCount` is called, `this.symbolCount` would increase.  I would suggest wrapping this in a check that it has not been called before, or resetting `this.symbolCount` to 0 at the beginning of the function.